packages/forma-36-website/src/pages/foundation/typography/design.mdx

```diff
@@ -0,0 +1,128 @@
+
+<br />
+
+## Font sizes
+Pixels units maintainence is rigid. Hence, we use *REM* units to define our font sizes.
+
+### What are REMs?
+REMs (root em) are simply a ratio of the root element font size. The root element here is the browser default font size 16px. (1rem = 16px) (2rem = 32px)
+
+Using rem units enables our system to scale up/down based on user's prefrence.
+
+
+<br />
+
+### Available Sizes
+
+<br />
+<SectionHeading>Section Heading</SectionHeading>
+
+*Section Heading* addresses the small blocks that requires extra context, to make it easier for audience to understand.
+
+Font size: **12px** - Line Height: **18px** - Weight: **600**
+
+<br />
+
+<Paragraph>Body</Paragraph>
+
+*Body* is the base font size for content. Used usually for paragraphs.
+
+Font size: **14px** - Line Height: **21px** - Weight: **500**
+
+<br />
+
+<Subheading>Subheading</Subheading>
+
+*Subheadings* are headings for nested blocks within your sections; Assuming you have a section with heading "Contact us", you want to group "Address" related elements in a block, that when *Subheading* comes in handy.
+
+ Font size: **16px** - Line Height: **24px** - Weight: **700**
+
+<br />
+
+<Heading>Heading</Heading>
+
+*Heading* is used to define sections in your solution, this is **cruical** for accessibility; Headings are used to define sections/blocks in your page, it guides users through your page/view.
+
+Similarly visually impaired people use screen readers, and screen readers find sections in your page/view based on your heading hierarchy.
+
+Font size: **21px** - Line Height: **31px** - Weight: **700**
+
+<br />
+
+<DisplayText>Display</DisplayText>
+
+Use *Display* on similar cases as Display Text Large.
+
+Font size: **28px** - Line Height: **35px** - Weight: **700**
+
+<br />
+
+<DisplayText size='large'>Display Large</DisplayText>
+
+*Display Large* is used to display text in special scenarios - use Display Text Large on empty states, promotional/featured items, etc.
+
+*Display Large* should not be used as a replacement for headings/page titles (use the Heading styles instead).
+
+Font size: **35px** - Line Height: **44px** - Weight: **700**
+
+<br />
+
+### Line height
+
+Line height are basically verticle wrappers around letters on each line. 
+
+Our line height is also defined based on **rem**, this forces the line height to correspond to the root font size as well.
+
+
+### Consistency
+![Line-height demonstration against 4px grid](./assets/line-height.png)
+
+To ensure each font type is represented properly, and the customer experience stays the same across different devices, 
+each font family members falls under the 4px base line grid.
+
+Now when different font types, sizes, and weights are stacked together, our typography system adapts flawlessly.
+
+<br />
+
+## Guidelines
+
+### Use Forma36 Sizes
+
+Don't define your own text sizes, use the available text styles from the sketch UI Kit. This ensures consistency across our solutions.
```

Comment:
To ensure consistency across user interfaces utilizing the Forma 36 design system, use available text styles from the Sketch UI kit.